mllib/src/main/scala/org/apache/spark/ml/clustering/GaussianMixture.scala

```diff
@@ -105,6 +108,15 @@ class GaussianMixtureModel private[ml] (
   def gaussians: Array[MultivariateGaussian] = parentModel.gaussians
 
   @Since("2.0.0")
```

Comment:
@jkbradley It make sense to use a natural class to represent ```gaussians```. I'm OK with providing a dataframe for now in Python. 
@wangmiao1981 Let's keep two variables: ```gaussians``` and ```gaussiansDF``` at Scala side. ```gaussiansDF``` is used for Python reference.